### DISCUSSION TITLE
Compatibility with HiFi-ESP32 boards?

### DISCUSSION THREAD

**Discussion by Drizzt321** [CarlosDerSeher/snapclient#107] [Q&A]
This is an awesome project! I had no idea, otherwise I probably would have done this long ago, instead of the RPI + Amp hats I mostly have.

I am looking for another endpoint or two, and came across [this set](https://www.tindie.com/products/sonocotta/hifi-esp32/) of ESP32 HiFi boards, some with DAC only, some with DAC+Amps. Seems likely that this project would be compatible with them, other than perhaps the higher powered amp boards based on [TAS5805M](https://www.ti.com/product/TAS5805M), since it sounds like you only support the TAS57XX line. 

Would you recommend the full dual core ESP-32? Or would the ESP-32-S3 single core work fine? And would the optional Ethernet module be supported as well?

**Comment by CarlosDerSeher** [comment]
Have a look at https://sonocotta.github.io/esparagus-snapclient/

**Reply by CarlosDerSeher** [reply]
Also TAS5805M should be supported

**Reply by Drizzt321** [reply]
Oh, totally didn't find that page. Thanks, that'll give me all I need to go on!
